pkg/plugin/clientmgmt/registry.go

```diff
@@ -92,6 +97,50 @@ func (r *registry) discoverPlugins(commands []string) error {
 			return err
 		}
 
+		// For backwards compatibility, some plugins will export both ItemSnapshotter plugins and
```

Comment:
I had two reasons.

1 - We'll need some way to restore from backups that have volume snapshots rather than item snapshots.  My thought was to continue to use the existing paths

2 - The plugin authors may want to have a single plugin that works with multiple versions of Velero.  Also, the vSphere BackupItemAction currently handles other resources in addition to PVCs.